exercises/practice/yacht/.meta/proof.ci.wren

```diff
@@ -0,0 +1,106 @@
+class Yacht {
+
+  static score(dice, category) {
```

Comment:
If this was a mentoring session, I'd have something to say about the length of this method ;)